bin/uc

```diff
@@ -29,7 +21,7 @@ run_cli_java_command() {
   # Print the quoted arguments for debugging
   # echo "Quoted arguments: $quoted_args_str"
   # echo "Running command: $CLI_JAVA_COMMAND"
-  eval $CLI_JAVA_COMMAND $@  || exit
+  eval "$CLI_JAVA_COMMAND" "$@"  || exit
```

Comment:
@abrassel I took a look at why the tutorial tests are failing, I think it's due to this line. Not sure why the `eval` was used in the first place but I think if you remove it and make the line `$CLI_JAVA_COMMAND "$@"  || exit` that should fix things. Can you give it a try? There might be a way to make the `eval` work but this now makes sense why all the args are being wrapped in quotes before being passed here.

Reply:
This seems right based on #263

Reply:
Unfortunately, `eval` seems to be required. Otherwise, we get this:
```
bin/uc: line 24: java -cp /Users/alexander.brassel/unitycatalog/examples/cli/target/classes:/Users/alexander.brassel/unitycatalog/server/target/classes:/Users/alexander.brassel/unitycatalog/clients/java/target/classes:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-tcnative-boringssl-static/2.0.65.Final/netty-tcnative-boringssl-static-2.0.65.Final-windows-x86_64.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/fasterxml/jackson/datatype/jackson-datatype-jsr310/2.17.0/jackson-datatype-jsr310-2.17.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/commons-cli/commons-cli/1.7.0/commons-cli-1.7.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-resolver-dns-native-macos/4.1.108.Final/netty-resolver-dns-native-macos-4.1.108.Final-osx-x86_64.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/delta/delta-kernel-api/3.2.0/delta-kernel-api-3.2.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-resolver-dns-classes-macos/4.1.108.Final/netty-resolver-dns-classes-macos-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/xerial/snappy/snappy-java/1.1.10.4/snappy-java-1.1.10.4.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/glassfish/jaxb/jaxb-core/4.0.2/jaxb-core-4.0.2.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/commons-pool/commons-pool/1.6/commons-pool-1.6.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/logging/log4j/log4j-api/2.23.1/log4j-api-2.23.1.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-tcnative-boringssl-static/2.0.65.Final/netty-tcnative-boringssl-static-2.0.65.Final-osx-aarch_64.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-resolver/4.1.108.Final/netty-resolver-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/roaringbitmap/RoaringBitmap/1.0.1/RoaringBitmap-1.0.1.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/vertx/vertx-web-client/4.3.5/vertx-web-client-4.3.5.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-codec/4.1.108.Final/netty-codec-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/jakarta/activation/jakarta.activation-api/2.1.3/jakarta.activation-api-2.1.3.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/parquet/parquet-jackson/1.12.3/parquet-jackson-1.12.3.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/micrometer/micrometer-observation/1.12.4/micrometer-observation-1.12.4.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/micrometer/micrometer-commons/1.12.4/micrometer-commons-1.12.4.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-transport-native-kqueue/4.1.108.Final/netty-transport-native-kqueue-4.1.108.Final-osx-x86_64.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/google/guava/failureaccess/1.0.1/failureaccess-1.0.1.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/sun/istack/istack-commons-runtime/4.1.1/istack-commons-runtime-4.1.1.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.17.0/jackson-databind-2.17.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/hibernate/orm/hibernate-core/6.5.0.Final/hibernate-core-6.5.0.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/hadoop/hadoop-client-api/3.4.0/hadoop-client-api-3.4.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-transport-native-unix-common/4.1.108.Final/netty-transport-native-unix-common-4.1.108.Final-osx-x86_64.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/httpcomponents/httpclient/4.5.14/httpclient-4.5.14.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-codec-dns/4.1.108.Final/netty-codec-dns-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-codec-http2/4.1.108.Final/netty-codec-http2-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/commons/commons-lang3/3.4/commons-lang3-3.4.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/fasterxml/jackson/datatype/jackson-datatype-jdk8/2.17.0/jackson-datatype-jdk8-2.17.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-core/1.12.728/aws-java-sdk-core-1.12.728.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/commons-logging/commons-logging/1.2/commons-logging-1.2.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/openapitools/jackson-databind-nullable/0.2.6/jackson-databind-nullable-0.2.6.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/commons-codec/commons-codec/1.15/commons-codec-1.15.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-tcnative-boringssl-static/2.0.65.Final/netty-tcnative-boringssl-static-2.0.65.Final-osx-x86_64.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/de/vandermeer/skb-interfaces/0.0.1/skb-interfaces-0.0.1.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/delta/delta-storage/3.2.0/delta-storage-3.2.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-transport-native-kqueue/4.1.108.Final/netty-transport-native-kqueue-4.1.108.Final-osx-aarch_64.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/httpcomponents/core5/httpcore5/5.2.4/httpcore5-5.2.4.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-resolver-dns/4.1.108.Final/netty-resolver-dns-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/h2database/h2/2.2.224/h2-2.2.224.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/github/ben-manes/caffeine/caffeine/2.9.3/caffeine-2.9.3.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-transport-classes-kqueue/4.1.108.Final/netty-transport-classes-kqueue-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/de/vandermeer/ascii-utf-themes/0.0.1/ascii-utf-themes-0.0.1.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/json/json/20240303/json-20240303.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/jakarta/inject/jakarta.inject-api/2.0.1/jakarta.inject-api-2.0.1.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/iceberg/iceberg-bundled-guava/1.5.2/iceberg-bundled-guava-1.5.2.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/vertx/vertx-web-common/4.3.5/vertx-web-common-4.3.5.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/vertx/vertx-auth-common/4.3.5/vertx-auth-common-4.3.5.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/hadoop/hadoop-client-runtime/3.4.0/hadoop-client-runtime-3.4.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/iceberg/iceberg-api/1.5.2/iceberg-api-1.5.2.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/net/bytebuddy/byte-buddy/1.14.15/byte-buddy-1.14.15.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/reactivestreams/reactive-streams/1.0.4/reactive-streams-1.0.4.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/iceberg/iceberg-common/1.5.2/iceberg-common-1.5.2.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/iceberg/iceberg-core/1.5.2/iceberg-core-1.5.2.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/httpcomponents/core5/httpcore5-h2/5.2.4/httpcore5-h2-5.2.4.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/logging/log4j/log4j-core/2.23.1/log4j-core-2.23.1.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/yetus/audience-annotations/0.13.0/audience-annotations-0.13.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-transport/4.1.108.Final/netty-transport-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/parquet/parquet-format-structures/1.12.3/parquet-format-structures-1.12.3.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/yaml/snakeyaml/2.2/snakeyaml-2.2.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/delta/delta-kernel-defaults/3.2.0/delta-kernel-defaults-3.2.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/jboss/logging/jboss-logging/3.5.0.Final/jboss-logging-3.5.0.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/smallrye/jandex/3.1.2/jandex-3.1.2.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/aayushatharva/brotli4j/brotli4j/1.16.0/brotli4j-1.16.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/slf4j/slf4j-api/1.7.36/slf4j-api-1.7.36.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/de/vandermeer/asciitable/0.3.2/asciitable-0.3.2.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-codec-haproxy/4.1.108.Final/netty-codec-haproxy-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/checkerframework/checker-qual/3.19.0/checker-qual-3.19.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-kms/1.12.728/aws-java-sdk-kms-1.12.728.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/parquet/parquet-column/1.12.3/parquet-column-1.12.3.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-tcnative-boringssl-static/2.0.65.Final/netty-tcnative-boringssl-static-2.0.65.Final-linux-aarch_64.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/jakarta/persistence/jakarta.persistence-api/3.1.0/jakarta.persistence-api-3.1.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/parquet/parquet-common/1.12.3/parquet-common-1.12.3.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/logging/log4j/log4j-slf4j-impl/2.23.1/log4j-slf4j-impl-2.23.1.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-handler/4.1.108.Final/netty-handler-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/github/luben/zstd-jni/1.5.0-1/zstd-jni-1.5.0-1.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/jakarta/xml/bind/jakarta.xml.bind-api/4.0.0/jakarta.xml.bind-api-4.0.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/vertx/vertx-uri-template/4.3.5/vertx-uri-template-4.3.5.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/glassfish/jaxb/txw2/4.0.2/txw2-4.0.2.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-common/4.1.108.Final/netty-common-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/antlr/antlr4-runtime/4.13.0/antlr4-runtime-4.13.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/vertx/vertx-core/4.3.5/vertx-core-4.3.5.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/httpcomponents/httpcore/4.4.16/httpcore-4.4.16.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/httpcomponents/client5/httpclient5/5.3.1/httpclient5-5.3.1.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-transport-native-unix-common/4.1.108.Final/netty-transport-native-unix-common-4.1.108.Final-linux-x86_64.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/google/errorprone/error_prone_annotations/2.10.0/error_prone_annotations-2.10.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/avro/avro/1.11.3/avro-1.11.3.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/antlr/ST4/4.0.8/ST4-4.0.8.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/fasterxml/classmate/1.5.1/classmate-1.5.1.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-core/2.17.0/jackson-core-2.17.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/vertx/vertx-bridge-common/4.3.5/vertx-bridge-common-4.3.5.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/latencyutils/LatencyUtils/2.0.3/LatencyUtils-2.0.3.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/joda-time/joda-time/2.12.7/joda-time-2.12.7.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.17.0/jackson-annotations-2.17.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-codec-http/4.1.108.Final/netty-codec-http-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-transport-native-unix-common/4.1.108.Final/netty-transport-native-unix-common-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/parquet/parquet-encoding/1.12.3/parquet-encoding-1.12.3.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/google/code/findbugs/jsr305/3.0.2/jsr305-3.0.2.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-buffer/4.1.108.Final/netty-buffer-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/hadoop/hadoop-aws/3.4.0/hadoop-aws-3.4.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-codec-socks/4.1.108.Final/netty-codec-socks-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/aayushatharva/brotli4j/service/1.16.0/service-1.16.0.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/google/guava/listenablefuture/9999.0-empty-to-avoid-conflict-with-guava/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/fasterxml/jackson/dataformat/jackson-dataformat-cbor/2.12.6/jackson-dataformat-cbor-2.12.6.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/hdrhistogram/HdrHistogram/2.1.12/HdrHistogram-2.1.12.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/software/amazon/awssdk/bundle/2.23.19/bundle-2.23.19.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/google/j2objc/j2objc-annotations/1.3/j2objc-annotations-1.3.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-tcnative-boringssl-static/2.0.65.Final/netty-tcnative-boringssl-static-2.0.65.Final-linux-x86_64.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/vertx/vertx-web/4.3.5/vertx-web-4.3.5.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/antlr/antlr-runtime/3.5.2/antlr-runtime-3.5.2.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/amazonaws/jmespath-java/1.12.728/jmespath-java-1.12.728.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-s3/1.12.728/aws-java-sdk-s3-1.12.728.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/de/vandermeer/char-translation/0.0.2/char-translation-0.0.2.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/wildfly/openssl/wildfly-openssl/1.1.3.Final/wildfly-openssl-1.1.3.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/hibernate/common/hibernate-commons-annotations/6.0.6.Final/hibernate-commons-annotations-6.0.6.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/airlift/aircompressor/0.26/aircompressor-0.26.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-transport-native-unix-common/4.1.108.Final/netty-transport-native-unix-common-4.1.108.Final-osx-aarch_64.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-handler-proxy/4.1.108.Final/netty-handler-proxy-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/linecorp/armeria/armeria/1.28.4/armeria-1.28.4.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-transport-native-epoll/4.1.108.Final/netty-transport-native-epoll-4.1.108.Final-linux-x86_64.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-transport-native-unix-common/4.1.108.Final/netty-transport-native-unix-common-4.1.108.Final-linux-aarch_64.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-transport-classes-epoll/4.1.108.Final/netty-transport-classes-epoll-4.1.108.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-resolver-dns-native-macos/4.1.108.Final/netty-resolver-dns-native-macos-4.1.108.Final-osx-aarch_64.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/jakarta/transaction/jakarta.transaction-api/2.0.1/jakarta.transaction-api-2.0.1.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/google/guava/guava/31.0.1-jre/guava-31.0.1-jre.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/antlr/antlr4/4.5.1/antlr4-4.5.1.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/commons/commons-compress/1.22/commons-compress-1.22.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-tcnative-classes/2.0.65.Final/netty-tcnative-classes-2.0.65.Final.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/netty/netty-transport-native-epoll/4.1.108.Final/netty-transport-native-epoll-4.1.108.Final-linux-aarch_64.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/apache/parquet/parquet-hadoop/1.12.3/parquet-hadoop-1.12.3.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/glassfish/jaxb/jaxb-runtime/4.0.2/jaxb-runtime-4.0.2.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/io/micrometer/micrometer-core/1.12.4/micrometer-core-1.12.4.jar:/Users/alexander.brassel/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/eclipse/angus/angus-activation/2.0.0/angus-activation-2.0.0.jar io.unitycatalog.cli.UnityCatalogCli: File name too long
```